src/allotropy/allotrope/exceptions.py

```diff
@@ -0,0 +1,2 @@
+class AllotropeConversionError(Exception):
```

Comment:
thoughts on putting this in `src/allotropy/` so the import would be `from allotropy.exceptions`?

(regardless, it's a lot better than `from allotropy.allotrope.allotrope`!)